apps/wallet/src/ui/app/pages/home/nft-transfer/index.tsx

```diff
@@ -79,25 +93,30 @@ function NftTransferPage() {
     const handleOnClearSubmitError = useCallback(() => {
         setSendError(null);
     }, []);
-    const { loading } = useObjectsState();
-    return (
-        <div className={cl(st.container, { 'items-center': loading })}>
-            <Loading loading={loading}>
-                {selectedNft && objectId && hasPublicTransfer(selectedNft) ? (
-                    <>
-                        <PageTitle
-                            title="Send NFT"
-                            back={`/nft-details?${new URLSearchParams({
-                                objectId,
-                            }).toString()}`}
-                        />
-                        <div className={st.content}>
-                            <NFTDisplayCard
-                                nftobj={selectedNft}
-                                wideView
-                                size="sm"
-                            />
 
+    const closeSendToken = useCallback(() => {
+        navigate('/');
+    }, [navigate]);
+
+    return (
+        <Overlay
+            showModal={showModal}
+            setShowModal={setShowModal}
+            title="Send NFT"
+            closeOverlay={closeSendToken}
```

Comment:
We could change a bit the component to accept only one method probably `setShowModal`, `closeOverlay` is redundant.